README.md

```diff
@@ -220,4 +225,12 @@ The size of the mounted rootfs can be specified with the `rootfs-size` parameter
 
 ## Kernel
 
-It is possible to replace the Linux image on which the tests are run and mount a [filesystem image](#image). More information on how to do it can be found [here](docs/Kernel.md).
+This action offers to replace the Linux image on which the tests are run and mount the [file system image](#image). More information on how to do it [here](docs/Kernel.md).
```

Comment:
```suggestion
It is possible to replace the Linux image on which the tests are run and mount the [file system image](#image). More information on how to do it can be found [in the 'Kernel' section of the docs](docs/Kernel.md).
```